Explorer/Assets/DCL/SDKComponents/Utils/FindAvatarUtils.cs

```diff
@@ -0,0 +1,73 @@
+using Arch.Core;
+using DCL.AvatarRendering.AvatarShape.Components;
+using DCL.AvatarRendering.AvatarShape.UnityInterface;
+using DCL.Multiplayer.Connections.Typing;
+using DCL.Multiplayer.Profiles.Tables;
+using System.Diagnostics;
+using UnityEngine;
+using Utility.Multithreading;
+
+namespace DCL.SDKComponents.Utils
+{
+    public static class FindAvatarUtils
+    {
+        private static readonly QueryDescription AVATAR_BASE_AND_SHAPE_QUERY = new QueryDescription().WithAll<AvatarBase, AvatarShapeComponent>();
+        private static readonly QueryDescription AVATAR_BASE_QUERY = new QueryDescription().WithAll<AvatarBase>();
+        public static IReadOnlyEntityParticipantTable? EntityParticipantTable;
```

Comment:
It's definitely an anti-pattern to have a static dependency. I wonder why you needed it so badly to break the rules

Reply:
It's having 1 line against having to modify 3+ files to inject an object proxy, but OK I'll do it that other way then

Reply:
Tackled at https://github.com/decentraland/unity-explorer/pull/3430/commits/8861ff81ea16779ee379fb547fd0353f842945fc

Reply:
Thanks